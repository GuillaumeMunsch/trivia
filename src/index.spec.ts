// eslint-disable-next-line import/no-extraneous-dependencies
import runGoldenMaster from "jest-golden-master";
import { Game } from ".";

describe("Trivia", () => {
  it("Create game", async () => {
    runGoldenMaster(async () => {
      // GIVEN

      // WHEN
      const game = new Game();
    });
  });

  it("Add 3 player", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();

      // WHEN
      game.add("Mathieu");
      game.add("Guillaume");
      game.add("Max");
    });
  });

  it("2 rolls of 1", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");

      // WHEN
      game.roll(1);
      game.roll(1);
    });
  });

  it("5 rolls of 1", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");

      // WHEN
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
    });
  });

  it("Ask all categories", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");

      // WHEN
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
      game.roll(1);
    });
  });

  it("Answer wrong and change player", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.add("Guillaume");
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);

      // WHEN
      game.wrongAnswer();
    });
  });

  it("Wrong answer, roll odd and out of bounds", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);
      game.wrongAnswer();

      // WHEN
      game.roll(7);
      game.roll(7);
    });
  });

  it("Wrong answer of last player and reset current player to first in list", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();

      // WHEN
      game.add("Mathieu");
      game.add("Guillaume");
      game.wrongAnswer();
    });
  });

  it("Correctly answered by in penality player and getting out of it", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();

      // WHEN
      game.add("Mathieu");
      game.wrongAnswer();
      game.roll(1);
      game.wasCorrectlyAnswered();
    });
  });

  it("Correctly answered by in penality player and getting out of it but not last player", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();

      // WHEN
      game.add("Mathieu");
      game.add("Guillaume");
      game.wrongAnswer();
      game.roll(1);
      game.wasCorrectlyAnswered();
    });
  });

  it("Correctly answered by in penality player and NOT getting out of it", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();

      // WHEN
      game.add("Mathieu");
      game.wrongAnswer();
      game.roll(2);
      game.wasCorrectlyAnswered();
    });
  });

  it("Correct answer, roll and out of bounds", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);
      game.wasCorrectlyAnswered();

      // WHEN
      game.roll(7);
      game.roll(7);
    });
  });

  it("Correct answer, roll and out of bounds", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);
      game.wrongAnswer();

      // WHEN
      game.roll(6);
      game.wasCorrectlyAnswered();
    });
  });

  it("Answer correctly", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);

      // WHEN
      game.wasCorrectlyAnswered();
    });
  });

  it("Answer wrongly", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);

      // WHEN
      game.wrongAnswer();
    });
  });

  it("Answer wrongly and then roll even", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);
      game.wrongAnswer();

      // WHEN
      game.roll(2);
    });
  });

  it("Answer wrongly and then roll odd", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);
      game.wrongAnswer();

      // WHEN
      game.roll(3);
    });
  });

  it("Answer wrongly and then roll odd and then answer correctly", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");
      game.roll(1);
      game.wrongAnswer();
      game.roll(3);

      // WHEN
      game.wasCorrectlyAnswered();
    });
  });

  it("Answer 6 times correctly", async () => {
    runGoldenMaster(async () => {
      // GIVEN
      const game = new Game();
      game.add("Mathieu");

      // WHEN
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
    });
  });

  it("My global scenario", async () => {
    runGoldenMaster(async () => {
      const game = new Game();
      game.add("Mathieu");
      game.add("Thomas");
      game.add("Cl√©ment");
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wrongAnswer();
      game.roll(1);
      game.wrongAnswer();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wrongAnswer();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
      game.roll(1);
      game.wasCorrectlyAnswered();
    });
  });
});