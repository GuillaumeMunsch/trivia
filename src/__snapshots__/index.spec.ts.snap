// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trivia 2 rolls of 1 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 2",
  "Asking question: Sports Question 0",
]
`;

exports[`Trivia 5 rolls of 1 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 2",
  "Asking question: Sports Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 3",
  "Asking question: Rock Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 4",
  "Asking question: Pop Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 5",
  "Asking question: Science Question 1",
]
`;

exports[`Trivia Add 3 player 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Adding player Guillaume to the game",
  "Number of players: 2",
  "Adding player Max to the game",
  "Number of players: 3",
]
`;

exports[`Trivia Answer 6 times correctly 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered correctly",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 2",
  "Asking question: Sports Question 0",
  "Player Mathieu answered correctly",
  "He's got 2 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 3",
  "Asking question: Rock Question 0",
  "Player Mathieu answered correctly",
  "He's got 3 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 4",
  "Asking question: Pop Question 0",
  "Player Mathieu answered correctly",
  "He's got 4 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 5",
  "Asking question: Science Question 1",
  "Player Mathieu answered correctly",
  "He's got 5 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 6",
  "Asking question: Sports Question 1",
  "Player Mathieu answered correctly",
  "He's got 6 points now",
  "Player Mathieu did win",
  "He is the winner",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Answer correctly 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered correctly",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Answer wrong and change player 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Adding player Guillaume to the game",
  "Number of players: 2",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered correctly",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Guillaume",
  "Player Guillaume rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 1",
  "Player Guillaume answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
]
`;

exports[`Trivia Answer wrongly 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
]
`;

exports[`Trivia Answer wrongly and then roll even 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 2",
  "While in penalityBox",
  "He won't get out of penality box",
]
`;

exports[`Trivia Answer wrongly and then roll even and then answer correctly 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 2",
  "While in penalityBox",
  "He won't get out of penality box",
  "Player Mathieu answered correctly",
  "He currently is in penality box",
  "And he's NOT getting out of it",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Answer wrongly and then roll odd 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 3",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 4",
  "Asking question: Pop Question 0",
]
`;

exports[`Trivia Answer wrongly and then roll odd and then answer correctly 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 3",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 4",
  "Asking question: Pop Question 0",
  "Player Mathieu answered correctly",
  "He currently is in penality box",
  "And he's getting out of it",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Ask all categories 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 2",
  "Asking question: Sports Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 3",
  "Asking question: Rock Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 4",
  "Asking question: Pop Question 0",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 5",
  "Asking question: Science Question 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 6",
  "Asking question: Sports Question 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 7",
  "Asking question: Rock Question 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 8",
  "Asking question: Pop Question 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 9",
  "Asking question: Science Question 2",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 10",
  "Asking question: Sports Question 2",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 11",
  "Asking question: Rock Question 2",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 0",
  "Asking question: Pop Question 2",
]
`;

exports[`Trivia Correct answer, roll and out of bounds 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered correctly",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 7",
  "While not being in penalityBox",
  "He moves to place 8",
  "Asking question: Pop Question 0",
  "Player Mathieu rolled 7",
  "While not being in penalityBox",
  "He moves to place 3",
  "Asking question: Rock Question 0",
]
`;

exports[`Trivia Correct answer, roll and out of bounds 2`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 6",
  "While in penalityBox",
  "He won't get out of penality box",
  "Player Mathieu answered correctly",
  "He currently is in penality box",
  "And he's NOT getting out of it",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Correctly answered by in penality player and NOT getting out of it 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 2",
  "While in penalityBox",
  "He won't get out of penality box",
  "Player Mathieu answered correctly",
  "He currently is in penality box",
  "And he's NOT getting out of it",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Correctly answered by in penality player and getting out of it 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 1",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered correctly",
  "He currently is in penality box",
  "And he's getting out of it",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Correctly answered by in penality player and getting out of it but not last player 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Adding player Guillaume to the game",
  "Number of players: 2",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Guillaume",
  "Player Guillaume rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Guillaume answered correctly",
  "He's got 1 points now",
  "Player Guillaume did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
]
`;

exports[`Trivia Create game 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
]
`;

exports[`Trivia My global scenario 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Adding player Thomas to the game",
  "Number of players: 2",
  "Adding player Clément to the game",
  "Number of players: 3",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered correctly",
  "He's got 1 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Thomas",
  "Player Thomas rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 1",
  "Player Thomas answered wrongly",
  "He is put in penality box",
  "The current player is now Clément",
  "Player Clément rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 2",
  "Player Clément answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 2",
  "Asking question: Sports Question 0",
  "Player Mathieu answered correctly",
  "He's got 2 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Thomas",
  "Player Thomas rolled 1",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 2",
  "Asking question: Sports Question 1",
  "Player Thomas answered wrongly",
  "He is put in penality box",
  "The current player is now Clément",
  "Player Clément rolled 1",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 2",
  "Asking question: Sports Question 2",
  "Player Clément answered correctly",
  "He currently is in penality box",
  "And he's getting out of it",
  "He's got 1 points now",
  "Player Clément did not win",
  "He isn't the winner",
  "The next player is player Mathieu",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 3",
  "Asking question: Rock Question 0",
  "Player Mathieu answered correctly",
  "He's got 3 points now",
  "Player Mathieu did not win",
  "He isn't the winner",
  "The next player is player Thomas",
  "Player Thomas rolled 1",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 3",
  "Asking question: Rock Question 1",
  "Player Thomas answered correctly",
  "He currently is in penality box",
  "And he's getting out of it",
  "He's got 1 points now",
  "Player Thomas did not win",
  "He isn't the winner",
  "The next player is player Clément",
]
`;

exports[`Trivia Wrong answer of last player and reset current player to first in list 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Adding player Guillaume to the game",
  "Number of players: 2",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Guillaume",
]
`;

exports[`Trivia Wrong answer, roll odd and out of bounds 1`] = `
Array [
  "Pop questions: 50",
  "Science questions: 50",
  "Sports questions: 50",
  "Rock questions: 50",
  "Adding player Mathieu to the game",
  "Number of players: 1",
  "Player Mathieu rolled 1",
  "While not being in penalityBox",
  "He moves to place 1",
  "Asking question: Science Question 0",
  "Player Mathieu answered wrongly",
  "He is put in penality box",
  "The current player is now Mathieu",
  "Player Mathieu rolled 7",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 8",
  "Asking question: Pop Question 0",
  "Player Mathieu rolled 7",
  "While in penalityBox",
  "He rolled an odd number",
  "He might get out of penality box",
  "He moves to place 3",
  "Asking question: Rock Question 0",
]
`;
